doc/Task.md

```diff
@@ -4,36 +4,38 @@
 - Don't use `'use strict';` in task files, tasks are always in strict mode
 - Tasks are placed in `application/tasks`
 - Task fields:
-  - `caption`
-  - `execute` allowed values: `single`, `server`, `instance`
-  - `time` (optional) defines certain time in UTC
-  - `interval` (optional) defines interval in milliseconds
-  - `run` is an async function to execute
+  - `caption` - task display name
+  - `execute` - execution type, allowed values: `single`, `server`, `instance`
+  - `time` (optional) - defines certain time in UTC
+  - `interval` (optional) - defines interval in milliseconds
+  - `timeout` (optional) - task execution timeout in milliseconds
+  - `run` - async function to execute
 - Task properties in runtime:
-  - `name` file name without extension used to access tasks
-  - `success` last execution status: `undefined`, `true`, `false`
-  - `error` last error or `null` (if success)
-  - `lastStart` last execution start time or `null` (never)
-  - `lastEnd` last execution end time or `null` (never)
-  - `executing` current status: `true`, `false`
-  - `active` is active in current process: `true`, `false`
-  - `count` executing count `number`
+  - `name` - file name without extension used to access tasks
+  - `success` - last execution status: `undefined`, `true`, `false`
+  - `error` - last error or `null` (if success)
+  - `lastStart` - last execution start time or `null` (never)
+  - `lastEnd` - last execution end time or `null` (never)
+  - `executing` - current status: `true`, `false`
+  - `active` - active in current process: `true`, `false`
```

Comment:
IMO It is not clear what is the difference between `executing` and `active` and what does `active` mean here.